Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -170,7 +170,8 @@ public String filterAlleStart() {
     }
 
     /**
-     * This method initializes the task list without any filter whenever the bean is created.
+     * This method initializes the task list without any filter whenever the
```

Comment:
are you using the right code formatter? 
formatting stuff shouldn't happen if the line is not touched.